jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-kops-aws.yaml

```diff
@@ -0,0 +1,67 @@
+- job-template:
+    name: 'kubernetes-e2e-kops-{suffix}'
+    node: '{jenkins_node}'
+    description: '{description} Test owner: {test-owner}.'
+    jenkins_node: 'e2e'
+    disabled: '{obj:disable_job}'
+    properties:
+        - build-discarder:
+            days-to-keep: 7
+    triggers:
+        - reverse:
+            jobs: '{trigger-job}'
+            result: success
+        - timed: '{cron-string}'
+    publishers:
+        - junit-publisher
+        - email-ext:
+            recipients: '{emails}'
+        - gcs-uploader
+    provider-env: |
+        export E2E_OPT="--deployment kops --kops $WORKSPACE/kops -kops-cluster {suffix}.test-aws.k8s.io -kops-state s3://k8s-kops-jenkins/"
+        export AWS_CONFIG_FILE=/workspace/.aws/credentials
+        # This is needed to be able to create PD from the e2e test
+        export AWS_SHARED_CREDENTIALS_FILE=/workspace/.aws/credentials
+        export KUBE_SSH_USER=admin
+    builders:
+        - activate-gce-service-account
+        - shell: |
+            {provider-env}
+            {job-env}
+            {post-env}
+            if ! curl -fsS --retry 3 -o "$WORKSPACE/kops.url" "https://storage.googleapis.com/kops-ci/bin/latest-ci.txt"; then
```

Comment:
The part that's really messy without doing this inline is setting the `NODEUP_URL`. I need to save both the NODEUP_URL export and the kops binary. I can do it `ssh-agent -c` style and have the binary spit out the necessary exports, but it's dirty.